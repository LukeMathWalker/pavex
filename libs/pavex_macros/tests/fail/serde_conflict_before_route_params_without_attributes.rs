#[derive(serde::Deserialize)]
#[pavex_macros::RouteParams]
struct MyStruct {
    field1: i32,
    field2: String,
}

fn main() {}
