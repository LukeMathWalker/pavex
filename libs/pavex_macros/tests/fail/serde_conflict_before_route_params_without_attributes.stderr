error[E0119]: conflicting implementations of trait `Deserialize<'_>` for type `MyStruct`
 --> tests/fail/serde_conflict_before_route_params_without_attributes.rs:2:1
  |
1 | #[derive(serde::Deserialize)]
  |          ------------------ first implementation here
2 | #[pavex_macros::RouteParams]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ conflicting implementation for `MyStruct`
  |
  = note: this error originates in the derive macro `serde::Deserialize` (in Nightly builds, run with -Z macro-backtrace for more info)
